src/ComponentPage.js

```diff
@@ -19,23 +21,41 @@ class ComponentPage extends Component {
     this.props.scrollTargetSetter(this.demoContentEl);
   }
 
+  scrollTop = () => {
+    // TODO: when React Tab issue is fixed, this can be removed.
```

Comment:
Is this irrelevant now? I think we removed React Tab from this PR?

Reply:
yup removed!